src/UI/Input/EventFormGUI.php

```diff
@@ -276,10 +276,11 @@ protected function initForm() {
 		$this->addItem($te);
 
 		$te = new ilTextInputGUI($this->txt(self::F_PRESENTERS), self::F_PRESENTERS);
-		$te->setRequired($this->schedule);
+		if (xoctConf::getConfig(xoctConf::F_PRESENTER_MANDATORY) || $this->schedule){
```

Comment:
Why do you check for $this->schedule? I'm not sure if all plugin adopters would agree to that.